python/lsst/ts/standardscripts/maintel/integration_tests/ccw_rotator_stress_test.py

```diff
@@ -0,0 +1,266 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["CcwRotatorStressTest"]
+
+import yaml
+import asyncio
+
+import numpy as np
+
+import astropy.units as u
+from astropy.coordinates import Angle
+
+from lsst.ts import salobj
+from lsst.ts.observatory.control.maintel.mtcs import MTCS, MTCSUsages
+
+np.random.seed(14)
+
+
+class CcwRotatorStressTest(salobj.BaseScript):
+    """Perform stress test between Camera Cable Wrap (CCW) and Rotator.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    * rotator_angle: before staring each sequence of random rotator angle.
+
+    * cycle: before each cycle of + and -.
+
+    * reset_rotator: after first test completes, before reseting the Rotator
+        position.
+
+    * delay: Before each "delay test".
+
+    **Details**
+
+    The test consists of:
+
+    * Start by slewing/tracking a circumpolar target with the rotator angle set
+      to be close to zero.
```

Comment:
Why not track in az/el instead? Advantages:
* It makes the test more predictable.
* It simplifies the code since you don't have to compute any kind of rotator offset.
* You can pick any convenient elevation, in case you want to study the effect of gravity. (I suggest making that a configuration parameter with a reasonable default).
* You can slew to the current azimuth, which will potentially speed up the test.